src/javascript/index.js

```diff
@@ -0,0 +1,91 @@
+import '../sass/styles.scss';
+
+import config from './config';
+
+window.onload = function() {
+  //draw_4();
+
+  let canvas = document.getElementById("canvas"),
+  ctx = canvas.getContext("2d");
+
+  ctx.fillStyle = "#e6e6e6";
+  ctx.fillRect(0, 0, 512, 512);
+
+  let s_4x4 = document.querySelector(".small");
+  s_4x4.addEventListener("click", draw_4);
+  let s_32x32 = document.querySelector(".medium");
+  s_32x32.addEventListener("click", draw_32)
+  let s_512x512 = document.querySelector(".large");
+  s_512x512.addEventListener("click", draw_512);
+}
+
+//4x4
+function draw_4() {  
+  let A = [["00BCD4", "FFEB3B","FFEB3B","00BCD4"],
```

Comment:
What is "A", try to use plain and straight forward names.

Reply:
I suppose we can move this "constant" out from function body, because each time we call this function we initialize this object.